test/accessor_tests.cc

```diff
@@ -336,7 +325,7 @@ namespace detail {
 				CHECK(aw.get_window_offset_in_buffer()[0] == 1);
 				CHECK(aw.get_buffer_range()[0] == 10);
 				CHECK(aw.get_window_offset_in_allocation()[0] <= 1);
-				CHECK(aw.get_allocation_range()[0] >= 6);
+				CHECK(aw.get_allocation_range()[0] >= 5);
```

Comment:
?

Reply:
This is a drive-by fix because the test fails with IDAG - if you look at the host task range you can see that the correct limit in dimension 0 is indeed `5`.